test/compilable/extra-files/ddocAny-postscript.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 
+source tools/common_funcs.sh
```

Comment:
Explicitly `source`ing for every `POST_SCRIPT` isn't pretty but they will be replaced anyway in the near future.